twill-yarn/src/test/java/org/apache/twill/filesystem/HDFSLocationTest.java

```diff
@@ -47,4 +52,14 @@ public static void finish() {
   protected LocationFactory createLocationFactory(String pathBase) throws Exception {
     return new HDFSLocationFactory(dfsCluster.getFileSystem(), pathBase);
   }
+
+  // TODO (TWILL-209): figure out how to make MiniDFSCluster enforce permissions
+  @Ignore
+  @Test(expected = Exception.class)
+  public void testPermissions() throws IOException {
+    // create a directory that does not permit anything
+    dfsCluster.getFileSystem().mkdir(new Path("/a"), FsPermission.valueOf("----------"));
```

Comment:
Oh ok, missed the Ignore annotation. It might be good to try and catch and have an Assert.fail at the end of the try block for the new tests that we are adding.